test/tokens/cpu.test.js

```diff
@@ -42,11 +42,14 @@ describe('cpu', () => {
     beforeEach(() => {
       contract = mockContract();
       cpuBalance = 10;
+      fetch.resetMocks();
+      fetch.mockResponses(mock([`${cpuBalance}.0000 CPU`]));
     });
 
     describe('when authorized', () => {
       test('returns', async () => {
         const result = await orejs.approveCpu(ORE_OWNER_ACCOUNT_NAME, ORE_TESTA_ACCOUNT_NAME, cpuBalance);
+        // expectFetch(`${ORE_NETWORK_URI}/v1/chain/get_currency_balance`);
```

Comment:
Can we remove this comment, if we're not going to have it working, for now?

Reply:
done